app/controllers/question.js

```diff
@@ -56,4 +58,47 @@ exports.getQuestionsWithPagination = async (req, res) => {
             }
         }).catch(() => res.status(401).json({ success: false, error: 'Ocorreu um erro ao buscar as questões.' }))
     }
+}
+
+const isCoordinator = async (user) => {
```

Comment:
Isso poderia ser reduzido para:
```
const isCoordinator = async user => !!(await Coordinator.findOne({where:{user_id:user.id}}))
```
`!!`, a dupla negação, é o "cast booleano" do javascript. Ele vai retornar um valor booleano para uma expressão dependendo se ela for falsy (null, undefined, string vazia, zero, false) ou truthy (todos outros valores)